djautotask/api_rest.py

```diff
@@ -468,3 +467,26 @@ def update_project(self, project, changed_fields):
 
     def _endpoint(self):
         return '{}{}'.format(self.api_base_url, self.POST_QUERY)
+
+
+class TicketChecklistItemsAPIClient(AutotaskAPIClient):
+    API = 'ChecklistItems'
```

Comment:
Oh I see. If making a request for all "TicketChecklistItems", the endpoint is `/V1.0/TicketChecklistItems/query` but create and update requests need `/V1.0/Tickets/{parentId}/ChecklistItems`. 

My GET requests are still failing though so I think this needs to be updated.

Reply:
To clarify, this is broken for me, but for the other comment the request query is being formatted properly

Reply:
Ok, got this fixed. Sorry about this! This happened because this is where I tried to use the `PARENT_API` class attribute and realized that it only had any effect on the `Task` api, which led me to dig into the AT API more deeply. I must have forgotten to fix it after I created issue 2142

Reply:
Looks good. Thanks!